package.json

```diff
@@ -13,7 +13,6 @@
     "postsemantic-release": "npm run coverage"
   },
   "nyc": {
-    "//": "don't let nyc instrument users-design-doc.js because it will be executed inside PouchDB",
```

Comment:
oh wow, good catch, how could did I miss that? I’d suggest combine both lines into one, that should work